docs/data/sdks/android-kotlin/index.md

```diff
@@ -78,14 +86,35 @@ You can configure the server zone when initializing the client for sending data
     )
     ```
 
+### Step 4: Send events
+
+```kotlin
+// Track a basic event
+amplitude.track("Button Clicked")
+// Track events with optional properties
+amplitude.track(
+  "Button Clicked",
+  mapOf("buttonColor" to "primary")
+)
+```
+
+Events tracked are buffered locally and flushed every 30 seconds. After calling track() in your app, you may wait several seconds to see data appear on the Amplitude website.
```

Comment:
```suggestion
Events tracked are buffered locally and flushed every 30 seconds. After calling `track()` in your app, it may take several seconds for event data to appear in Amplitude.
```
Do my edits change the meaning? Edited to be a bit more clear about what the user should expect.

Reply:
Your edits don't change the meaning at all. I used ChatGPT to polish the sentence and got the similar edits as yours! 

Resolve this as it has been changed in quickstart guide